### ISSUE TITLE
Setup scan symbologies from CaptureHelper?

### ISSUE THREAD

**Issue by chbeer** [SocketMobile/swift-package-capturesdk#1]
<img width="823" alt="image" src="https://user-images.githubusercontent.com/121761/184625280-bb097bfb-3597-4a70-b76a-13a0889e119b.png">

Can we somehow enable and disable scan symbologies from CaptureHelper? We need the DuraScan 740 to ignore 1D Barcode and just scan QR-Codes.

**Comment by socketPM** [comment]
@chbeer yes, you can refer to the documentation here:https://docs.socketmobile.com/capture/ios/en/latest/topicsActivityLevel.html#enabling-or-disabling-a-barcode-symbology
We don't have a way to identify 1d or 2d. So you will have to disable all the 1d types individually and only enable QR Code.

**Comment by cyrille-socket** [comment]
Closed as there's no activity.
